cmd/registry/cmd/label.go

```diff
@@ -29,157 +29,67 @@ import (
 	"google.golang.org/genproto/protobuf/field_mask"
 )
 
-var labelFilter string
-var labelOverwrite bool
-var labelsToSet map[string]string
-var labelsToClear []string
+var labelKeyFilter string
+var labelKeyOverwrite bool
+var labelKeysToSet map[string]string
+var labelKeysToClear []string
+
+const labelFieldName = "labels"
+const labelCommandName = "label"
+
+var labelCmd = labelCommand()
```

Comment:
It didn't at the time, but I think it does now - see the latest iteration of the label.go and annotate.go declarations and the corresponding tests - I found during testing that values of certain flags would "stick" when the command was invoked multiple times from tests (particularly for the optional boolean "overwrite" flag). A handler constructor allows us to construct multiple instances of a command handler for use in tests and also moves us closer to being able to have a constructor for the root command (which would allow us to handle multiple commands in parallel, once all the global flags were removed)